InnerEye/ML/model_training.py

```diff
@@ -131,8 +127,15 @@ def create_lightning_trainer(container: LightningContainer,
     num_gpus = container.num_gpus_per_node
     effective_num_gpus = num_gpus * num_nodes
     # Accelerator should be "ddp" when running large models in AzureML (when using DDP_spawn, we get out of GPU memory).
-    # For unit tests, only "ddp_spawn" works
-    accelerator = "ddp" if effective_num_gpus > 1 else None
+    if effective_num_gpus > 1:
+        accelerator: Optional[str] = "ddp"
+        # Initialize the DDP plugin with find_unused_parameters=False by default. If True (default), it prints out
```

Comment:
the comment is a bit confusing for me, the default seems to be indicated 2 times with opposite values. Does this should be interpreted as suggestion to set pl_find_unused_parameters=False in the container? if yes I would slightly change as below. 
```suggestion
        # Initialize the DDP plugin with find_unused_parameters=False. If True (default), it prints out
```
Also, for my own understanding, where the user is setting up this default value for the container? maybe a comment there would help too?